control-plane-configs/cp3.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This looks like we should move to another data structure for ip pools. Maybe somthing like this would work:
ip_pools: {
  1: {
    pool: 192.168.21.1-255,
    net:  192.168.21.0/24
   },
  2: {
    pool: 192.168.22.1-255,
    net:  192.168.22.0/24
   },
...
}

If we stick with 192.168.XXX.0/24, there might be an even easier solution by just providing the range {} for XX and then creating all networking by iterating over this.